frontend/public/components/dashboard/inventory-card/status-group.ts

```diff
@@ -1,7 +1,14 @@
 export enum InventoryStatusGroup {
-  OK = 'OK',
   WARN = 'WARN',
   ERROR = 'ERROR',
   PROGRESS = 'PROGRESS',
   NOT_MAPPED = 'NOT_MAPPED',
+  UNKNOWN = 'UNKNOWN',
 }
+
+export const inventoryStatusGroupPriority = {
```

Comment:
When specifying `priority` value in the plugin, can we import this and do something like

```ts
inventoryStatusGroupPriority[InventoryStatusGroup.PROGRESS] + some_offset
```

instead of hard-coded numbers everywhere? This also creates a link to base priority values and therefore improves the understanding of extension's impact.

Reply:
sounds good, using offest values now

Reply:
since I removed at at the end..this is irrelevant now